web/src/user_topics_ui.js

```diff
@@ -17,7 +17,7 @@ export function handle_topic_updates(user_topic_event) {
 
     stream_list.update_streams_sidebar();
     unread_ui.update_unread_counts();
-    message_lists.current.update_muting_and_rerender();
+    message_lists.current?.update_muting_and_rerender();
```

Comment:
Should this one be a loop over `all_rendered_message_lists`?

Reply:
Yeah.